eq-author/src/App/sharing/SharePageContent/UserSearch.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import reactStringReplace from "react-string-replace";
+import PropTypes from "prop-types";
+import Downshift from "downshift";
+
+import Button from "components/buttons/Button";
+
+import {
+  SearchContainer,
+  SearchInput,
+  SearchResults,
+  SearchResult,
+  SearchResultName,
+  SearchResultEmail,
+  SearchField,
+  Highlight,
+  SpacedFlexContainer,
+} from "../styles/UserSearch";
+
+const propTypes = {
+  UserSearch: {
+    users: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string,
+        name: PropTypes.string,
+        email: PropTypes.string,
+      })
+    ),
+    onUserSelect: PropTypes.func.isRequired,
+  },
+};
+
+const highlighSearchTerm = (...args) =>
+  reactStringReplace(...args, (match, i) => (
+    <Highlight key={i}>{match}</Highlight>
+  ));
+
+const UserSearch = ({ users, onUserSelect }) => {
+  const [user, setUser] = useState(null);
+
+  const addUser = target => {
+    setUser(target);
+  };
+
+  return (
+    <>
+      <Downshift
+        initialIsOpen={false}
+        onSelect={addUser}
+        itemToString={user => (user ? user.name || user.email : "")}
+      >
+        {({
+          getInputProps,
+          getItemProps,
+          getMenuProps,
+          isOpen,
+          inputValue,
+          highlightedIndex,
+        }) => (
+          <div>
+            <SearchContainer>
+              <SearchField>
+                <SpacedFlexContainer>
+                  <SearchInput
+                    {...getInputProps()}
+                    placeholder="Search people by name or email address"
```

Comment:
The placeholder text isn't on the design. As a rule of thumb, we don't placeholder text as it's inaccessible.